### PR TITLE
Etredal/add model version

### PR DESCRIPTION
**Pull request by etredal**
Adding additional .txt file to container images so this can be accessed

### CODE REVIEW

File: presets/inference/llama2-chat/inference_api.py [hunk 1]
```diff
@@ -191,6 +191,13 @@ def get_metrics():
         except Exception as e:
             return {"error": str(e)}
 
+    @app_main.get("/version")
+    def health_check():
+        with open("/workspace/llama/model_name.txt", "r") as f:
+            model_name = f.read()
+
+        return {"version": model_name}
```
Comment (ishaansehgal99) [inline presets/inference/llama2-chat/inference_api.py:199]: Currently version here will just print model_name, we also need the version (e.g. 0.0.3)

File: presets/inference/llama2-completion/inference_api.py [hunk 2]
```diff
@@ -180,6 +180,13 @@ def get_metrics():
         except Exception as e:
             return {"error": str(e)}
 
+    @app_main.get("/version")
+    def health_check():
+        with open("/workspace/tfs/model_name.txt", "r") as f:
+            model_name = f.read()
+
+        return {"version": model_name}
```
Comment (ishaansehgal99) [inline presets/inference/llama2-completion/inference_api.py:188]: same as above

File: presets/inference/text-generation/inference_api.py [hunk 3]
```diff
@@ -210,6 +210,13 @@ def get_metrics():
     except Exception as e:
         return {"error": str(e)}
 
+@app.get("/version")
+def health_check():
+    with open("/workspace/tfs/model_name.txt", "r") as f:
+        model_name = f.read()
+
+    return {"version": model_name}
```
Comment (ishaansehgal99) [inline presets/inference/text-generation/inference_api.py:218]: same as above

File: presets/inference/llama2-chat/inference_api.py [hunk 4]
```diff
@@ -191,6 +191,13 @@ def get_metrics():
         except Exception as e:
             return {"error": str(e)}
 
+    @app_main.get("/version")
+    def health_check():
+        with open("/workspace/llama/model_name.txt", "r") as f:
```
Comment (ishaansehgal99) [inline presets/inference/llama2-chat/inference_api.py:196]: move filename here to be a constant up top

**Comment by ishaansehgal99** [comment]
One more file needs to be edited:` docker-job-template.yaml` need to change the docker build command to use the new `IMAGE_NAME` you added

**Comment by etredal** [comment]
@microsoft-github-policy-service agree

**Pull request closed without merging**